### PR TITLE
Group classes by day

### PR DESCRIPTION
**Pull request by zoey97wang**
sorry for delay, still try to find a good way to sort data by class in day

connects #12

### CODE REVIEW

File: app/src/main/java/com/njitdev/sa_android/classschedule/ClassScheduleActivity.java [hunk 1]
```diff
@@ -1,33 +1,13 @@
-/*
```
Comment (yunzhu-li) [inline app/src/main/java/com/njitdev/sa_android/classschedule/ClassScheduleActivity.java]: Why is the header gone?

File: app/src/main/java/com/njitdev/sa_android/classschedule/ClassScheduleActivity.java [hunk 2]
```diff
@@ -63,49 +44,109 @@ protected void onCreate(Bundle savedInstanceState) {
     // TODO: Group by day
     private void populateClassScheduleList() {
         mClassScheduleList = new LinkedList<>();
-        for (int i = 0; i < mClassSchedule.size(); i++) {
-            for (int j = 0; j < mClassSchedule.get(i).size(); j++) {
-                mClassScheduleList.add(mClassSchedule.get(i).get(j));
+        //TO DO AUG 20;
```
Comment (yunzhu-li) [inline app/src/main/java/com/njitdev/sa_android/classschedule/ClassScheduleActivity.java]: I think we can use the conventional `// TODO: ...` style for `todo`s

File: app/src/main/java/com/njitdev/sa_android/classschedule/ClassScheduleActivity.java [hunk 3]
```diff
@@ -41,9 +21,9 @@
 
 public class ClassScheduleActivity extends AppCompatActivity {
 
-    private ClassScheduleAdapter mAdapter;
+    //  private ClassScheduleAdapter mAdapter;
```
Comment (yunzhu-li) [inline app/src/main/java/com/njitdev/sa_android/classschedule/ClassScheduleActivity.java]: We can remove it if it's not used anymore

File: app/src/main/java/com/njitdev/sa_android/classschedule/ClassScheduleActivity.java [hunk 4]
```diff
@@ -63,49 +44,109 @@ protected void onCreate(Bundle savedInstanceState) {
     // TODO: Group by day
     private void populateClassScheduleList() {
         mClassScheduleList = new LinkedList<>();
-        for (int i = 0; i < mClassSchedule.size(); i++) {
-            for (int j = 0; j < mClassSchedule.get(i).size(); j++) {
-                mClassScheduleList.add(mClassSchedule.get(i).get(j));
+        //TO DO AUG 20;
+        //populateClassList by classified day of week;
+        List<List<ClassSchedule>> temp = new LinkedList<>();
+        for (int i = 0; i < 5; i++) {
```
Comment (yunzhu-li) [inline app/src/main/java/com/njitdev/sa_android/classschedule/ClassScheduleActivity.java]: Some students have class on weekends

**Comment by yunzhu-li** [comment]
No worries. This looks good, thanks!

I think there were a few issues, I fixed them already so this is ready to merge and we can move faster towards launch, you can go through the [changes](https://github.com/njitdev/sa-android/pull/41/commits/2a9bdad0c246472d6a634df3e9da73825634d442), basically:

- The [file header](https://github.com/njitdev/sa-android/pull/41/commits/ae2014cffd8191e60ad72d7880453796b853e521#diff-26dcec6f3804accdc625a838aaa3a8a8L1) was gone.

- Any reason to use `LinkedList`? We are [fetching objects by their index](https://github.com/njitdev/sa-android/pull/41/commits/ae2014cffd8191e60ad72d7880453796b853e521#diff-26dcec6f3804accdc625a838aaa3a8a8R64) so isn't `ArrayList` better suited?

- We can use the conventional `// TODO: ...` style for `todo`s and [comments not used](https://github.com/njitdev/sa-android/pull/41/commits/ae2014cffd8191e60ad72d7880453796b853e521#diff-26dcec6f3804accdc625a838aaa3a8a8R47) can be removed.

- For sorting classes in a day, we can [sort them by the `classes_in_day` field](https://github.com/njitdev/sa-android/pull/41/commits/2a9bdad0c246472d6a634df3e9da73825634d442#diff-bf2b5b4b8a4583ea780aacc350305cb9R240).

- I created [`mSectionedClassScheduleList`](https://github.com/njitdev/sa-android/pull/41/commits/2a9bdad0c246472d6a634df3e9da73825634d442#diff-26dcec6f3804accdc625a838aaa3a8a8R43) to allow week selection later.

- Renamed class `ClassScheduleAdapterwithTitle` to [camel case](https://en.wikipedia.org/wiki/Camel_case) `ClassScheduleAdapterWithTitle`.

**Comment by zoey97wang** [comment]
Yes, I agree ArrayList works better than LinkedList in our case, thank u.

**Pull request merged**